webpack/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint import/no-unresolved: [2, { ignore: [foremanReact/*] }] */
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/extensions */
+import componentRegistry from 'foremanReact/components/componentRegistry';
+import { registerReducer } from 'foremanReact/common/MountingService';
+import reducers from './src/reducers';
+import PluginTemplate from './src/PluginName';
+
+// register reducers
+Object.entries(reducers).forEach(([key, reducer]) =>
+  registerReducer(key, reducer)
+);
+
+// register components for erb mounting
+componentRegistry.register({
+  name: 'PluginTemplate',
+  type: PluginTemplate,
+});
```

Comment:
can you create another `component` file that contains pairs of `{ name, type }`
and here make a `forEach` loop, same as the `reducers` are being registered

Reply:
doesn't it overkill for a very small amount of reducers, also the reducer registering might be unclear for newcomers?

Reply:
I think that the registries will be very easy to use, as they won't even need to touch this file
and just put all of their component in one spot without calling each time `componentRegistry.register()`